lists/eg/eg-cr.json

```diff
@@ -1,46 +1,48 @@
 {
-    "links": {
-        "wikipedia": null,
-        "opencorporates": null
-    },
-    "sector": null,
-    "url": "http://www.gafi.gov.eg/ar/onestopshop/onestop.aspx",
-    "name": {
-        "en": "General Authority for Free Zones and Investment (GAFI), Commercial Registry",
-        "local": ""
-    },
-    "coverage": [
-        "EG"
+  "name": {
+    "en": "Commercial Registry",
```

Comment:
It's good to add the country to the title, when it isn't obvious. So: "Commercial Registry, Egypt"

Reply:
@kd-ods thanks for checking the new registration agency. I made the 2 required changes.